common/djangoapps/student/tests/test_admin_views.py

```diff
@@ -324,15 +324,26 @@ def setUp(self):
         """Setup."""
         super(LoginFailuresAdminTest, self).setUp()
         self.client.login(username=self.user.username, password='test')
-        user = UserFactory.create()
+        self.user2 = UserFactory.create()
         LoginFailures.objects.create(user=self.user, failure_count=10, lockout_until=datetime.datetime.now())
-        LoginFailures.objects.create(user=user, failure_count=2)
+        LoginFailures.objects.create(user=self.user2, failure_count=2)
 
     def tearDown(self):
         """Tear Down."""
         super(LoginFailuresAdminTest, self).tearDown()
         LoginFailures.objects.all().delete()
 
+    def test_unicode_username(self):
+        """
+        Test if `__str__` method behaves correctly for unicode username.
+        It shouldn't raise `TypeError`.
+        """
+        try:
+            str(LoginFailures.objects.get(user=self.user))
+            str(LoginFailures.objects.get(user=self.user2))
+        except TypeError as e:
```

Comment:
There's no reason to catch exceptions here.  Let them go, and the test will fail.

Reply:
Sure thing, removed.